### ISSUE TITLE
Persistence load balancer -UDP

### ISSUE THREAD

**Issue by ricardodeizeppe** [yyyar/gobetween#340]
Is there any mechanism that allows persistence of NODEs? I would like to use SIP and UDP, but for that, persistence is required.

Thanks,

**Comment by illarion** [comment]
Yes, it is implemented. Please refer to https://github.com/yyyar/gobetween/wiki/Protocols#udp
In 90% of the cases, you'll need to specify client_idle_timeout or backend_idle_timeout so that gobetween will be able to distinguish "sessions"
